### PR TITLE
add doc for environment variables

### PR DESCRIPTION
**Pull request by khassel**
introduced with https://github.com/MichMich/MagicMirror/pull/3032, related to https://github.com/MichMich/MagicMirror/issues/1756

### CODE REVIEW

**Comment by rejas** [comment]
Maybe we should split that into a seperate page like configuration/advanced ?

File: configuration/introduction.md [hunk 1]
```diff
@@ -23,3 +23,176 @@ The following properties can be configured:
...
+```bash
+export MY_ADDRESS=localhost
+export MYPORT=8080
+export MY_HTTPS=false
+```
+
+#### Real world example of `config.js.template`
+
+```js
+var config = {
```
Comment (KristjanESPERANTO) [inline configuration/introduction.md:90]: var -> let

Comment (khassel) [reply configuration/introduction.md]: thanks ... may I should update my config ;)

fixed.

**Comment by khassel** [comment]
> Maybe we should split that into a seperate page like configuration/advanced ?

see no sense to split this, it is part of configuration and the site is not to long

**Review by rejas — APPROVED** [review]

**Pull request merged**